.github/workflows/nanopub.yml

```diff
@@ -26,10 +26,7 @@ jobs:
       - name: Lint with flake8
         run: |
           pip install flake8
-          # stop the build if there are Python syntax errors or undefined names
-          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-          # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+          flake8 . --count --show-source --statistics
```

Comment:
I moved configuration to `tox.ini` file. That makes it easier to run it locally using the same settings. (maybe also handy for your CONTRIBUTING related changes)

Reply:
Nice, yes that does. I'll open a new issue to change it.

Reply:
Issue #74